app/models/agents/trigger_agent.rb

```diff
@@ -8,7 +8,7 @@ class TriggerAgent < Agent
     description <<-MD
       The Trigger Agent will watch for a specific value in an Event payload.
 
-      The `rules` array contains hashes of `path`, `value`, and `type`.  The `path` value is a dotted path through a hash in [JSONPaths](http://goessner.net/articles/JsonPath/) syntax.
+      The `rules` array contains hashes of `path`, `value`, and `type`.  The `path` value is a dotted path through a hash in [JSONPaths](http://goessner.net/articles/JsonPath/) syntax. For simple messages, this is the name of the field - for example simply body_text for a body_text event.
```

Comment:
I agree this is useful to clarify for someone who hasn't used JSONPath.

Maybe "For simple events, this is usually just the name of the field you want, like 'text' for the text key of the event." ?